app/models/lettings_log.rb

```diff
@@ -435,13 +435,29 @@ def self.to_csv(user = nil)
     Csv::LettingsLogCsvService.new(user).to_csv
   end
 
+  def validation_beds
+    if needstype == 2
+      0
+    else
+      beds.nil? ? nil : [beds, 4].min
```

Comment:
Beds is at most 4 in the rent range data - Jemma is currently checking that 4 beds in the data means 4 or greater in the property but this is the assumption we have agreed makes sense at the moment

Reply:
This is now in, ready to merge again